src/Symfony/Component/Clock/DatePoint.php

```diff
@@ -66,6 +66,30 @@ public static function createFromMutable(\DateTime $object): static
         return parent::createFromMutable($object);
     }
 
+    public static function createFromTimestamp(int|float $timestamp): static
+    {
+        if (\PHP_VERSION_ID >= 80400) {
+            return parent::createFromTimestamp($timestamp);
+        }
+
+        if (\is_int($timestamp)) {
+            return (new static())->setTimestamp($timestamp);
+        }
+
+        if (!is_finite($timestamp) || $timestamp > \PHP_INT_MAX + 1.0 || $timestamp < \PHP_INT_MIN) {
```

Comment:
Why is the `+1` part necessary?

Reply:
Reading the code below doesn’t the comparison have to be changed from `›` to `›=`?

Reply:
Yes, probably. Let me change it.